api_docs/include/rest-endpoints.md

```diff
@@ -116,6 +116,8 @@
 * [Reorder custom profile fields](/api/reorder-custom-profile-fields)
 * [Create a custom profile field](/api/create-custom-profile-field)
 * [Update realm-level defaults of user settings](/api/update-realm-user-settings-defaults)
+* [Get all public data exports](/api/get-realm-exports)
+* [Create a public data export](/api/export-realm)
```

Comment:
Just a small note that we're probably going to want to edit this label once we add support for other types of data exports in the API.